### DISCUSSION TITLE
Languages not shown correctly

### DISCUSSION THREAD

**Discussion by quotentiroler** [community/community#69357] [Repositories]
### Select Topic Area

Bug

### Body

![image](https://github.com/community/community/assets/40643627/996f9bbb-4b5c-42bb-ba51-d5c7c15c08df)

My repository has 52 Java files and 5 HTML files, but it shows the Languages like this.

**Comment by ghost** [comment]


**Reply by quotentiroler** [reply]
Thanks for your detailed response.

However, my repository consists of 52 Java files that contain lots of code, written by myself. Also, the repository contains 5 HTML that I created, but they are not that big.

It should be more like 95% Java. And I am pretty sure that last time I checked, it was showing like this.
